### PR TITLE
adding boolean like or dislike to response

### PR DESCRIPTION
**Pull request by marinaRomanchuk**

### CODE REVIEW

File: instagram_own_version/settings.py [hunk 1]
```diff
@@ -62,8 +63,12 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "corsheaders.middleware.CorsMiddleware",
```
Comment (revimi) [inline instagram_own_version/settings.py]: Let's move CORS related stuff it to another PR

File: posts/views.py [hunk 2]
```diff
@@ -52,6 +52,12 @@ def get(self, request, pk: int):
             "dislike_count": LikeDislike.objects.filter(
                 post_id=pk, is_like=False
             ).count(),
+            "like": LikeDislike.objects.filter(
```
Comment (revimi) [inline posts/views.py]: Not sure, but probably smth like `has_liked` will be clearer

File: posts/views.py [hunk 3]
```diff
@@ -52,6 +52,12 @@ def get(self, request, pk: int):
...
+                post_id=pk, user=request.user, is_like=True
+            ).exists(),
+            "dislike": LikeDislike.objects.filter(
```
Comment (revimi) [inline posts/views.py]: Not sure, but probably smth like `has_disliked` will be clearer

Comment (marinaRomanchuk) [reply posts/views.py]: Changed to has_disliked

File: posts/views.py (hunk 2 above)
Comment (marinaRomanchuk) [reply posts/views.py]: Changed to `has_liked`

File: instagram_own_version/settings.py (hunk 1 above)
Comment (marinaRomanchuk) [reply instagram_own_version/settings.py]: Removed

**Review by revimi — APPROVED** [review]
